docs/use-events.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Find Events in the Catalog
+title: Event Catalog
 ---
 # About the Catalog
 
-The [Event Catalog](https://hub.lighthouse.va.gov) is the single source of truth for all events currently published on the Event Bus. The Event Catalog lives on the [Lighthouse Hub](https://hub.lighthouse.va.gov), and you will need to be on the VA network to access it.
+The [Event Catalog](https://hub.lighthouse.va.gov) is the single source of truth for all events currently published on the Event Bus. The Event Catalog lives on [CODE VA](https://hub.lighthouse.va.gov) (formerly called the Lighthouse Hub), and you will need to be on the VA network to access it.
 
```

Comment:
Change link to https://code.va.gov/

Reply:
I will change this when code.va.gov is the production URL. Marking as resolved.